modin/pandas/groupby.py

```diff
@@ -171,10 +173,85 @@ def idxmax(self):
     def ndim(self):
         return 2  # ndim is always 2 for DataFrames
 
-    def shift(self, periods=1, freq=None, axis=0):
-        return self._default_to_pandas(
-            lambda df: df.shift(periods=periods, freq=freq, axis=axis)
-        )
+    def shift(self, periods=1, freq=None, axis=0, fill_value=None):
+        """
```

Comment:
We don't need to add docstrings for public methods manually after [this commit](https://github.com/modin-project/modin/commit/0a3bea4a719470690abd7673f9e81a34a64a4de8) in order to avoid inconsistencies with pandas. You can find the explanation/description [here](https://github.com/modin-project/modin/pull/2196#issue-496793251) and [here](https://github.com/modin-project/modin/commit/0a3bea4a719470690abd7673f9e81a34a64a4de8#diff-506c226c0b2753ce3bbf33ea6c4de22796376d982b483a4d9bbbb3e3e6561a9dR15), for instance.

Reply:
Docstrings was removed.